src/main/java/com/eternalcode/parcellockers/parcel/repository/ParcelWrapper.java

```diff
@@ -0,0 +1,63 @@
+package com.eternalcode.parcellockers.parcel.repository;
+
+import com.eternalcode.parcellockers.parcel.Parcel;
+import com.eternalcode.parcellockers.parcel.ParcelSize;
+import com.j256.ormlite.field.DatabaseField;
+import com.j256.ormlite.table.DatabaseTable;
+
+import java.util.Set;
+import java.util.UUID;
+
+@DatabaseTable(tableName = "parcels")
+class ParcelWrapper {
+
+    @DatabaseField(id = true, columnName = "uuid")
```

Comment:
I almost didn't notice that this field was set as id.
```suggestion
    @DatabaseField(columnName = "uuid", id = true)
```